security/pkg/pki/ca/controller/secret.go

```diff
@@ -207,9 +210,17 @@ func (sc *SecretController) upsertSecret(saName, saNamespace string) {
 		PrivateKeyID: key,
 		RootCertID:   rootCert,
 	}
-	_, err = sc.core.Secrets(saNamespace).Create(secret)
+
+	// We retry several times when create secret.
```

Comment:
Mention this is to reduce transient network issue?

Reply:
Done.